js/packages/phoenix-client/src/constants.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * Supported model providers
+ */
+export type PhoenixModelProvider =
+  | "OPENAI"
+  | "AZURE_OPENAI"
+  | "ANTHROPIC"
+  | "GEMINI";
+
+/**
+ * The role of a chat message
+ */
+export type PhoenixChatMessageRole = "user" | "system" | "ai" | "tool";
```

Comment:
should this be in types?

Also wondering if we nix the phoenix prefix for clarity of code inside here. NBD